contrib/completion/bash/docker

```diff
@@ -593,7 +593,8 @@ __docker_append_to_completions() {
 # The result is cached for the duration of one invocation of bash completion.
 __docker_fetch_info() {
 	if [ -z "$info_fetched" ] ; then
-		read -r client_experimental server_experimental server_os < <(__docker_q version -f '{{.Client.Experimental}} {{.Server.Experimental}} {{.Server.Os}}')
+		# prevent bash substitution error in cygwin : /dev/fd/62 : No such file or directory error
```

Comment:
I don't think we need this comment here. It will not be useful for readers once this PR is merged.
It is sufficient to mention the rationale behind this change in the commit message.